_data/simple-icons.json

```diff
@@ -16792,6 +16792,11 @@
             "source": "https://venmo.com/about/brand",
             "guidelines": "https://venmo.com/about/brand"
         },
+	    {
+	        "title": "Vencord",
+	        "hex": "8E71D4",
+	        "source": "https://raw.githubusercontent.com/Vencord/Vesktop/main/build/icon.png?format=webp&quality=lossless&width=662&height=662"
```

Comment:
This does not follow https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines.